statistics/handle/updatetest/update_test.go

```diff
@@ -446,6 +446,8 @@ func TestAutoUpdate(t *testing.T) {
 		tableInfo = tbl.Meta()
 		h.HandleAutoAnalyze(is)
 		require.NoError(t, h.Update(is))
+		testKit.MustExec("explain select * from t where a > 'a'")
+		require.NoError(t, h.LoadNeededHistograms())
```

Comment:
After turning on `lite-init-stats` by default, we need to use `LoadNeededHistograms` to load index stats when lease is not zero.